runtime/klee-libc/fortify-klibc.c

```diff
@@ -0,0 +1,37 @@
+//===-- fortify-klibc.c ---------------------------------------------------===//
+//
+//                     The KLEE Symbolic Virtual Machine
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+/* Fortified versions of the libc functions defined in the klee-libc library */
+
+#include "klee/klee.h"
+
+#include <string.h>
+
+void *__mempcpy_chk(void *dest, const void *src, size_t len, size_t destlen) {
```

Comment:
This function is already part of `fortify-fs.c` and if `klibc` is linked `Freestanding` will be linked as well.